### PR TITLE
Fix editLink plugin + add plugin example

### LINKED ISSUE #224: Create Link doesn't work
When trying to insert link, I select text, click insert link, fill url, nothing changes. Console shows no errors.
Last working commit is 3dd6cc1c76ce048ad7c05d3037fa80c321844a0e and editlink plugin is a bit broken too.
I tried it in Opera, Vivaldi, Chrome, Firefox.

### PR DESCRIPTION
**Pull request by ra100**
-  fix EditLink
-  createLink override - Inset Link button now inserts new link, and modifies existing link
-  unlink override - if no text is selected and cursor is "inside" link, selection expands and removes whole link
-  editlink.html example
-  fixes #224

### CODE REVIEW

**Comment by Alex-D** [comment]
Since the code is less heavy, I think to add it in Trumbowyg core. I need to test it asap.

Thanks for your involvement in this project :)

File: plugins/editlink/trumbowyg.editlink.js [hunk 1]
```diff
@@ -12,63 +12,40 @@
...
-                                range.selectNode(sel.focusNode);
-                                sel.addRange(range);
-                                t.saveSelection();
-                                node = t.selection.commonAncestorContainer;
-                            } else {
-                                t.saveSelection();
-                                node = t.selection.commonAncestorContainer.parentNode;
-                            }
+                        var node = sel.extentNode;
+                        while (node.nodeName != 'A' && node.nodeName != 'DIV') {
```
Comment (Alex-D) [inline plugins/editlink/trumbowyg.editlink.js]: Use `!==` insteed of `!=`

File: plugins/editlink/trumbowyg.editlink.js [hunk 2]
```diff
@@ -12,63 +12,40 @@
...
-                editLink: {
+                createLink: {
                     fn: function (params, tbw) {
                         var t = tbw,
                             url = '',
                             title = '',
                             target = '_blank',
                             node = '',
-                            edit = false;
+                            edit = false,
```
Comment (Alex-D) [inline plugins/editlink/trumbowyg.editlink.js]: This variable is not anymore used

**Comment by Alex-D** [comment]
I merge it, but I need to change some things to be OK with jshint and maybe fix your workaround if it's possible :)

**Comment by ra100** [comment]
Seems I'll have to tweak my jshint and beautify settings, so there is no need to fix formatting in the future.

**Comment by Alex-D** [comment]
As you cas see on `develop` I've merge your plugin in core. I've also fix some bugs in my code and fix your workaroud (it was my fault).

Thanks for your PR!

**Comment by ra100** [comment]
Awesome.

**Pull request merged**